File: mybatis-plus-join-test/test-springboot3-jdk17/src/main/java/com/yulichang/test/springboot3jdk17/mapper/MyBaseMapper.java
package com.leheyue.test.springboot3jdk17.mapper;

import com.github.leheyue.base.MPJBaseMapper;
import org.apache.ibatis.annotations.Mapper;

import java.io.Serializable;
import java.util.List;

@Mapper
@SuppressWarnings("unused")
public interface MyBaseMapper<T extends Serializable> extends MPJBaseMapper<T> {

    int insertBatchSomeColumn(List<T> entityList);
}

File: mybatis-plus-join-test/test-kotlin/src/main/kotlin/com/github/yulichang/test/kt/mapper/MyBaseMapper.kt
package com.github.leheyue.test.kt.mapper

import com.github.leheyue.base.MPJBaseMapper
import org.apache.ibatis.annotations.Mapper

@Mapper
interface MyBaseMapper<T> : MPJBaseMapper<T> {
    fun insertBatchSomeColumn(entityList: List<T>?): Int
}
